docs/lab-1/README.md

```diff
@@ -1,3 +1,9 @@
+---
+title: Using Prompt Engineering for your Application
```

Comment:
Prompt Engineering for your Application --> Prompt Engineering for Identifying the LLM for your Application

Reply:
The challenge is that the "title" only has so many text slots.

![image](https://github.com/IBM/watsonx-genai-lab/assets/810824/a5b54c17-8dd6-449e-ab46-2224d26456ae)

I think the title I proposed here was too long now that I look at it.

Reply:
How about: Identify LLM for your App?